space.celestiaproject.Celestia.metainfo.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2022 Mattia Verga <mattia.verga@proton.me> -->
+<component type="desktop-application">
+    <id>space.celestiaproject.Celestia</id>
+    <metadata_license>MIT</metadata_license>
+    <project_license>GPL-2.0+</project_license>
+    <name>Celestia</name>
+    <developer_name>Celestia Project</developer_name>
+    <developer id="celestia.space">
+      <name>Celestia Project</name>
+    </developer>
+    <summary>Explore the universe</summary>
+    <description>
+    <p>
+      Celestia provides photo-realistic, real-time, three-dimensional viewing of
+      the Solar System, the Galaxy and the Universe.
+    </p>
+    <p>
+      It is an easy to use, freely-distributed, multi-platform, open source,
+      software package which has become a valuable tool for astronomy education.
+      Used in homes, schools, museums and planetariums around the world, it also
+      is used as a visualization tool by space mission designers.
+    </p>
+  </description>
+  <launchable type="desktop-id">space.celestiaproject.Celestia.desktop</launchable>
+  <url type="homepage">https://celestiaproject.space</url>
+
+  <content_rating type="oars-1.0">
+  </content_rating>
```

Comment:
```suggestion
  <content_rating type="oars-1.1" />
```

https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#oars-information